easybuild/easyblocks/generic/mesonninja.py

```diff
@@ -49,7 +51,10 @@ def extra_options(extra_vars=None):
         """Define extra easyconfig parameters specific to MesonNinja."""
         extra_vars = EasyBlock.extra_options(extra_vars)
         extra_vars.update({
+            'build_dir': [None, "build_dir to pass to meson", CUSTOM],
+            'build_cmd': [DEFAULT_BUILD_CMD, "Build command to use", CUSTOM],
```

Comment:
This breaks the CMakeNinja EasyBlock which doesn't call this method and hence has "make" as the build_cmd